java/http/java-http-producer/src/main/java/HttpProducer.java

```diff
@@ -122,4 +124,3 @@ private void send() {
         }
     }
 }
-
```

Comment:
why removing this line?

Reply:
Put back one empty line. Should be correct now.